gradle.properties

```diff
@@ -189,5 +189,29 @@ jfigletVersion=0.0.8
 
 embeddedRedisVersion=0.6
 
+#webjars versions
```

Comment:
Master. 5.0 will get its own branch. I'll try to get things updated today or this week if all goes well unless you beat me to it :)